### PR TITLE
Added platform arg to docker build

### PR DESCRIPTION
**Pull request by drpayyne**
Hi @jlandure, thanks a lot for your work!

Here's my PR to open discussion for publishing multi-arch images:

https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/ has a quick guide on what I propose. The `docker buildx way`. I have been testing this out and using this extensively to publish multi-arch image to multiple registries. 

Suggestion - what about using GitHub Actions to publish multi-arch multi-registry images? Here's my repo which implements it this way - https://github.com/drpayyne/docker-php. There's a single workflow file for each package and I build for both `linux/arm64` and `linux/amd64` and publish it to both DockerHub and GitHub Packages. 

This is a draft PR which I intend to take forward in your preferred way of action. Please let me know how we can do this. From a quick overview, only the Deno Dockerfile needs a little modification to get it to work multi-platform. We need to pick the appropriate Deno download URL based on the target architecture. A preview of how it would be is implemented by me for another repo at https://github.com/markshust/docker-magento/pull/516/files

### CODE REVIEW

**Comment by Ferror** [comment]
Go ahead and use mine pipeline.

```yaml
   
name: Build & Publish Docker Image

on:
    push:

jobs:
    build:
        runs-on: ubuntu-latest
        steps:
            -
                name: checkout code
                uses: actions/checkout@v2
            -
                name: Login to Docker Hub
                uses: docker/login-action@v1
                with:
                    username: ${{ secrets.DOCKER_HUB_USERNAME }}
                    password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
            -
                name: install buildx
                id: buildx
                uses: crazy-max/ghaction-docker-buildx@v1
                with:
                    buildx-version: latest
            -
                name: build the image
                run: |
                    docker buildx build \
                        --push \
                        --tag ${{ secrets.DOCKER_HUB_USERNAME }}/REPO-NAME:TAG \
                        --platform linux/amd64,linux/arm/v7,linux/arm64 .
```

**Comment by Ferror** [comment]
@jlandure any updates regarding linux/arm64 image?

File: hooks/build [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-docker image build --build-arg VCS_REF=`git rev-parse --short HEAD` --build-arg BUILD_DATE=`date -u +”%Y-%m-%dT%H:%M:%SZ”` -t $IMAGE_NAME .
+docker image build --build-arg VCS_REF=`git rev-parse --short HEAD` --platform linux/arm64,linux/amd64 --build-arg BUILD_DATE=`date -u +”%Y-%m-%dT%H:%M:%SZ”` -t $IMAGE_NAME .
```
Comment (zigarn) [inline hooks/build:4]: Shouldn't it be?
```suggestion
docker buildx build --build-arg VCS_REF=`git rev-parse --short HEAD` --platform linux/arm64,linux/amd64 --build-arg BUILD_DATE=`date -u +”%Y-%m-%dT%H:%M:%SZ”` -t $IMAGE_NAME .
```

**Comment by hgwood** [comment]
Hi @drpayyne! Your change has been included in #201. Thank you very much.

**Comment by drpayyne** [comment]
Thank you @hgwood and @zigarn! Apologies that I missed the PR comment notification.

**Pull request closed without merging**